benchmark-reporter/package.json

```diff
@@ -0,0 +1,50 @@
+{
+  "name": "benchmark-reporter",
+  "version": "1.0.0",
+  "description": "A Probot app",
+  "author": "Joseph Agnelli <jagnelli@uber.com>",
+  "license": "ISC",
+  "repository": "https://github.com//my-first-app.git",
+  "homepage": "https://github.com//my-first-app",
+  "bugs": "https://github.com//my-first-app/issues",
+  "keywords": [
+    "probot",
+    "github",
+    "probot-app"
+  ],
+  "scripts": {
+    "dev": "nodemon",
+    "start": "probot run ./index.js",
+    "lint": "standard --fix",
```

Comment:
I think we'll probably want to add repo-wide linting later (as opposed to per-package linting)

```suggestion
```